src/LogBackend.cxx

```diff
@@ -9,11 +9,12 @@
 #include "Version.h"
 #include "config.h"
 
+#include <fmt/chrono.h>
 #include <cassert>
-
-#include <stdio.h>
-#include <string.h>
-#include <time.h>
+#include <cstdio>
+#include <cstring>
```

Comment:
Why change this line? It's not related to the rest of this commit, is it? Sneaking in unrelated changes makes a PR very hard to read, and there isn't even a good reason to do so.